Sources/PointFree/Account/Change.swift

```diff
@@ -31,10 +22,22 @@ let subscriptionChangeMiddleware: Middleware<
       )
 )
 
+private let requireUserAndPricingAndSeats
+  : MT<Tuple2<User?, Pricing?>, Tuple3<User, Int, Pricing>>
```

Comment:
Using an `MT` type as a shorthand for describing middleware transformers...makes it slightly easier to break this stuff up.

Reply:
I've actually got typealiases for this I've been using in other parts:

https://github.com/pointfreeco/pointfreeco/blob/master/Sources/PointFree/TODO.swift#L20-L21